pkg/resources/database_acceptance_test.go

```diff
@@ -725,7 +725,7 @@ func TestAcc_Database_IntParameter(t *testing.T) {
 			{
 				PreConfig: func() {
 					param := acc.TestClient().Parameter.ShowAccountParameter(t, sdk.AccountParameterDataRetentionTimeInDays)
-					require.Equal(t, "", string(param.Level))
+					require.Equal(t, string(sdk.ParameterTypeAccount), string(param.Level))
```

Comment:
nope. There is one test that is not cleaning this up correctly but we should not have it set up (@sfc-gh-jcieslak   has its on his TODO list to find out which of the tests breaks the se tup: we tried unsettling this param, but after the test run it's there again). Please revert.

Reply:
Done.